Add implementation paragraph (#121)
anitvam/pps-bunny: doc/report.md

## Implementazione
Nei seguenti paragrafi ciascun componente descriverà per quali parti è responsabile dell'implementazione.
### Baiardi
Inizialmente mi sono dedicata all'implementazione dell'interfaccia grafica, in particolare ho definito:
* la struttura base dell'applicazione
* il movimento dei coniglietti nell'ambiente di simulazione 
* il pannello di scelta delle mutazioni da introdurre
* i pannelli visualizzati al termine della simulazione

Ho disegnato tutti i coniglietti con le possibili combinazioni di mutazioni in collaborazione con Spadoni e successivamente ho definito un singleton contenente la `Enumeration BunnyImage` con tutte le `Image` relative ai coniglietti, questa enumeration è particolarmente importante in quanto, grazie al metodo `bunnyToImage`, riesce a convertire l'istanza del model di un coniglietto nell'equivalente immagine ad esso associata. 

> Successivamente mi sono dedicata all'implementazione di `Environment` insieme a Spadoni, anche per quanto riguarda la visualizzazione grafica del clima. 

Reviewer: ```suggestion
Successivamente mi sono dedicata all'implementazione di `Environment` insieme a Spadoni e in autonomia alla visualizzazione grafica del clima. 
```

> Successivamente mi sono dedicata all'implementazione di `Environment` insieme a Spadoni, della cui parte di clima ho altrettanto implementato la visualizzazione grafica. 

Reviewer 2: ```suggestion
Successivamente mi sono dedicata all'implementazione di `Environment` insieme a Spadoni, anche per quanto riguarda la visualizzazione grafica del clima. 
```
